index.html

```diff
@@ -232,6 +232,42 @@ <h3> <img src="./img/Rectangle 57.png" alt="points" width="10%"> skills</h3>
 
   
 </section>
+<!-- Contact form -->
+<section id="form-section">
+  <div class="form-container">
+    <div class="form-intro">
+      <div>
+        <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+      </div>
+    </div>
+    <div class="form-data">
+      <form action="https://formspree.io/f/xlekyljn" method="POST">
+          <input type="text" class="first-input" placeholder="First Name" maxlength="30" required>
+          <input type="text"  class="second-input"  placeholder="Last Name" maxlength="30" required> <br>
+          <input  type="email"  class="third-input"  id="input4"  placeholder="Email Address"  required> <br>
+          <textarea type="text" class="fourth-input"  maxlength="500" name="message"  placeholder="Write me something..." required ></textarea> <br>
+        <button type="submit" class="form-btn">Get in touch</button>
+      </form>
+    </div>
+  </div>
+</section>
+
+
+<hr class="footer-hr">
+<!-- footer section -->
+
+
+<footer>
+  <div>
+    <ul class="foter-list">
+      <li><a href="#"><img src="./img/footer/1.png" alt="angel"></a></li>
+      <li><a href="#"><img src="./img/footer/2.png" alt="github"></a></li>
+      <li><a href="#"><img src="./img/footer/3.png" alt="medium"></a></li>
+      <li><a href="#"><img src="./img/footer/4.png" alt="twitter"></a></li>
+      <li><a href="#"><img src="./img/footer/5.png" alt="vector"></a></li>
+    </ul>
+  </div>
```

Comment:
- ***Given Design***
![image](https://github.com/Xghayor/Portfolio/assets/60083437/28056ea6-5420-415f-a95b-4b7cd7010e3d)

- For icons, kindly consider the things below to match the given design:
- [ ] Give more margin between the icons.
- [ ] Give more padding/margin between `hr` tag and `footer`
- [ ] Maintain the serial of icons as per the given design.